docs/api/api_docs.yaml

```
@@ -976,27 +976,111 @@ paths:
           type: string
         in: path
         required: true
+        description: 'The attachment name. This value must be URL encoded. For example, if the attachment name is `blob_/avatar`, the path component passed to the URL should be `blob_%2Favatar` (tested with [URLEncoder](https://www.urlencoder.org/)).'
     get:
       responses:
         '200':
-          description: OK
+          description: Found attachment successfully.
+          headers:
+            Etag:
+              schema:
+                type: string
+              description: 'The attachment digest. Does not get set when request `meta=true`. '
+        '206':
+          description: Partial attachment content returned
+        '404':
+          $ref: '#/components/responses/Not-found'
+        '416':
+          description: Requested range exceeds content length
       tags:
         - Admin
         - Public
+      summary: Get an attachment from a document
+      description: |-
+        This request retrieves a file attachment associated with the document. 
+
+        The raw data of the associated attachment is returned (just as if you were accessing a static file). The `Content-Type` response header is the same content type set when the document attachment was added to the database. The `Content-Disposition` response header will be set instead if the content type is vulnerable to a phishing attack, causing the browser to download the file instead of display it (unless overriden by by database config option `serve_insecure_attachment_types`).
+
+        If the `meta` query parameter is set then the response will be in JSON with the additional metadata tags.
+      parameters:
+        - $ref: '#/components/parameters/rev'
```

Missing request parameters

```suggestion
        - $ref: '#/components/parameters/rev'
        - schema:
            type: boolean
            default: 'true'
          in: query
          name: content_encoding
          description: Set to false to disable the `Content-Encoding` response header.
        - schema:
            type: string
          in: header
          name: Range
          description: 'RFC-2616 bytes range header. E.g: bytes=123-456'
```